### PR TITLE
RWA-1138: Fix worktypes not being parsed correctly

### PR DESCRIPTION
**Pull request by alexbaeza**
### Change description ###

Fix worktype list not being parsed correctly due to leading spaces

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by serdaroguzhanakin — DISMISSED** [review]

File: src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/WorkTypesControllerTest.java [hunk 1]
```diff
@@ -164,6 +164,40 @@ void should_return_a_valid_work_type_list_when_user_has_work_types() throws Exce
         );
     }
 
+    @Test
+    void should_return_a_valid_work_type_list_when_user_has_work_types_leading_spaces() throws Exception {
```
Comment (BenNzewi) [inline src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/WorkTypesControllerTest.java:168]: 👍 Nice one.

File: src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/WorkTypesService.java [hunk 2]
```diff
@@ -58,9 +59,14 @@ public WorkTypesService(CFTWorkTypeDatabaseService cftWorkTypeDatabaseService) {
         Set<String> roleWorkTypes = new HashSet<>();
 
         for (RoleAssignment roleAssignment : accessControlResponse.getRoleAssignments()) {
-            String assignedWorkedList = roleAssignment.getAttributes().get(WORK_TYPES.value());
-            if (assignedWorkedList != null) {
-                roleWorkTypes.addAll(asList(assignedWorkedList.split(",")));
+            if (roleAssignment.getAttributes() != null) {
```
Comment (BenNzewi) [inline src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/WorkTypesService.java:62]: can we cover by unit test when this is null?
